xarray/core/rolling.py

```diff
@@ -871,14 +873,48 @@ def wrapped_func(self, **kwargs):
 
         return wrapped_func
 
+    def reduce(self, func: Callable, **kwargs):
+        """Reduce the items in this group by applying `func` along some
```

Comment:
```suggestion
        """Reduce the items in this group by applying ``func`` along some
```
I think docstrings are in rst format?

Reply:
I went back and forth on this, and eventually opted to keep things as they were.  This is consistent with the other docstrings for [rolling](http://xarray.pydata.org/en/stable/generated/xarray.core.rolling.DataArrayRolling.html) and [coarsening](http://xarray.pydata.org/en/stable/generated/xarray.core.rolling.DataArrayCoarsen.html#xarray.core.rolling.DataArrayCoarsen) objects.  I don't have a strong opinion though.

Reply:
One backtick seems to render as italics and two as code block. In text form I also like the single backticks more. I am fine to leave as is.

https://xray--4939.org.readthedocs.build/en/4939/generated/xarray.core.rolling.DataArrayCoarsen.reduce.html#xarray.core.rolling.DataArrayCoarsen.reduce

https://xray--4939.org.readthedocs.build/en/4939/generated/xarray.core.weighted.DataArrayWeighted.mean.html